mdbenchmark/mdengines/gromacs.py

```diff
@@ -46,8 +49,8 @@ def parse_ns_day(fh):
     for line in lines:
         if 'Performance' in line:
             return float(line.split()[1])
-    warnings.warn(UserWarning, "No Performance data found.")
-    return 0
+
+    return 0.0
```

Comment:
why did you change this? The normal 0 worked just fine.

Reply:
The docstring says it returns a float, so I figured we should stick to it and also return a float in the default case.

Reply:
Returning `0` or `0.0` is equivalent in python2/3. The type is implicitly upcast in a operation with a float. Only in python 2 the division by an integer is problematic since that returns an int. But 0 divided by anything is also 0. So for 0, it doesn't make a difference.

Reply:
But as I have written above returning a `np.nan` is actually a better decision.